test/utils/docker/centos7/Dockerfile

```diff
@@ -17,15 +17,20 @@ RUN yum clean all && \
     bzip2 \
     dbus-python \
     file \
+    gcc \
     git \
     iproute \
+    libffi \
+    libffi-devel \
     make \
     mariadb-server \
     mercurial \
     MySQL-python \
     openssh-clients \
     openssh-server \
+    openssl-devel \
     python-coverage \
+    python-devel \
     python-httplib2 \
     python-jinja2 \
     python-keyczar \
```

Comment:
I believe every docker image except centos6 will have a python{2,3,}-cryptography package so it should just be centos6 that needs to be able to compile cryptography from a source tarball.